subt/docker/robotika/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ link_directories(
 ## Declare a C++ executable
 add_executable(subt_seed_node src/subt_seed_node.cc)
 add_dependencies(subt_seed_node ${catkin_EXPORTED_TARGETS})
-target_link_libraries(subt_seed_node ${catkin_LIBRARIES} -lzmq)
+target_link_libraries(subt_seed_node ${catkin_LIBRARIES} -L /usr/lib/x86_64-linux-gnu -lzmq)
```

Comment:
This definitely does not look right. If the path is not already preset, it most likely is not the correct path. For compiling x86_64 targets this path should be preset. Are you sure that is really the triplet you are compiling for?

Reply:
the strange thing for me is that without any path it does not link:
```
[100%] Linking CXX executable /home/developer/subt_solution/devel/lib/subt_seed/subt_seed_node
/usr/bin/ld: CMakeFiles/subt_seed_node.dir/src/subt_seed_node.cc.o: undefined reference to symbol 'zmq_recv'
/usr/lib/x86_64-linux-gnu/libzmq.so.5: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
subt_seed/CMakeFiles/subt_seed_node.dir/build.make:149: recipe for target '/home/developer/subt_solution/devel/lib/subt_seed/
subt_seed_node' failed
```
and with this path it reports:
```
ESC[91mWARNING: Target "subt_seed_node" requests linking to directory "/usr/lib/x86_64-linux-gnu".  Targets may link only to libraries.  CMake is dropping the item.
ESC[0mESC[91mWARNING: Target "subt_seed_node" requests linking to directory "/usr/lib/x86_64-linux-gnu".  Targets may link only to libraries.  CMake is dropping the item.
ESC[0m-- Generating done
```
but then it links it correctly. Here I needed any solution so suggestion how to fix the build is very welcome, thanks.

Reply:
What you are after is probably https://cmake.org/cmake/help/latest/command/link_directories.html - that should allow you to set the search path. What are the steps to test building this?

Reply:
```
cd subt/docker
./build.bash robotika
```

Reply:
The conclusion was that the change is not needed, the code builds fine without it.